style.css

```diff
@@ -611,3 +611,98 @@ form div {
   justify-content: start;
   align-items: initial;
 }
+
+/* This part is only for the popup_windows */
+.popup-container {
+  width: 100%;
+  height: 100%;
+  top: 0;
+  position: fixed;
+  background: rgba(27, 23, 102, 0.99);
+  display: none;
+  align-items: center;
+  justify-content: center;
+}
+
+.popup {
+  width: 90%;
+  max-width: 1108px;
+  height: auto;
+  border: 2px solid white;
+  padding: 24px;
+  background: rgba(255, 255, 255, 0.1);
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+}
```

Comment:
- [ ] Kindly fix the styling of the popup, because the height is overlapping and I can not see the close button and title of the modal. Here is the image of the style you did.
![](https://res.cloudinary.com/dflkwvvnz/image/upload/v1631080020/scrnli_9_8_2021_10-44-37_AM_lcklkx.png)

- As you saw, it is not working in all screen sizes, please fix the height 😊.